dockerize/create_nginx_conf.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+echo "What is your servers FQDN? "
+read FQDN
+echo "What is your servers IP? "
+read IP_ADDR
+
+cat <<EOF > nginx/b2share.conf
+server {
+        listen 80;
```

Comment:
We can not allow unencrypted traffic. Sometimes the web UI sends session confidential information to the server.